### DISCUSSION TITLE
Welcome to the Hyperapp discussions page! 🦄

### DISCUSSION THREAD

**Discussion by jorgebucaran** [jorgebucaran/hyperapp#998] [General]
# Hey!

GitHub has made _Discussions_ available, which can be used as an accessible, public forum for project discussions and questions. Here you can post for things such as:

- Ask questions and get answers
- Get help and feedback for your code using Hyperapp
- Share tips & tricks
- Stay up-to-date on announcements

Be sure to post in the relevant discussion category if applicable. Bugs and features should continue to be reported using the [issue tracker](https://github.com/jorgebucaran/hyperapp/issues). If you aren't comfortable posting publicly, you can always find me on Twitter @jorgebucaran, but I do encourage you to post here instead so that people can find the same answers you got in the future.

**Comment by zaceno** [comment]
Discussions seem awesome! Issues worked, but it was always a shame that they hid away valuable discussion that wasn’t strictly related to issues. Looking forward to seeing what comes up here.
